components/WeaveTransfer/index.tsx

```diff
@@ -77,33 +61,34 @@ const WeaveTransfer = () => {
 
     const othentInstance = await Othent({ API_ID: 'd7a29242f7fdede654171a0d3fd25163' });
 
-    let user_details 
+    let user_details
+
     if (JSON.parse(localStorage.getItem('othentUserDetails'))) {
       user_details = await othentInstance.userDetails()
     } else {
       // @ts-ignore
       user_details = await othentInstance.logIn()
       localStorage.setItem('othentUserDetails', JSON.stringify(user_details));
     }
-    
+
     if (user_details.message === 'new user created') {
       alert('New Othent account created! Please re send the file')
       setLoading(false)
     } else {
 
       const signedArweaveTransaction = await othentInstance.signTransactionArweave({
-        othentFunction: 'uploadData', 
+        othentFunction: 'uploadData',
         data: file,
         tags: [ {name: 'Content-Type', value: file.type} ]
       });
-  
+
       const transaction = await othentInstance.sendTransactionArweave(signedArweaveTransaction);
-    
+
       const formData = new FormData();
       formData.append("transaction_id", transaction.transactionId);
       formData.append("sendToEmail", sendToEmail);
       formData.append("sendFromEmail", user_details.email);
-    
+
       fetch('https://server.othent.io/weavetransfer', {
         method: 'POST',
         body: formData,
```

Comment:
We should update this code to use the new version of the SDK

Reply:
I created a [new issue](https://github.com/Othent/landing-new/issues/7) for this as I don't want to put much time into this site as it's likely going away soon. Once there's some clarity around that, I'll see what to do with WeaveTransfer (e.g. move it to the playground repo).